sysl2/sysl/visitor.go

```diff
@@ -337,6 +342,13 @@ func (v *SequenceDiagramVisitor) visitEndpoint(e *EndpointElement) error {
 	isCronSender := e.senderPatterns.Contains("cron")
 	needsInt := !(isHuman || isHumanSender || isCronSender) && sender != agent
 
+	if attr, exists := app.GetAttrs()[v.groupby]; exists {
```

Comment:
this code should not execute when v.groupby = "".
should only build group by if groupby string is non-empty

Reply:
Taken care.